neurodemo/channelparam.py

```diff
@@ -57,12 +59,12 @@ def __init__(self, ion):
         ]
         self.ion.Erev = self.Nernst(ion)
         self.ion_params.append(dict(name="Erev", type='float', value=self.ion.Erev, suffix='V', siPrefix=True, readonly=True))
-        pt.parameterTypes.SimpleParameter.__init__(self, name=name, type='bool', 
+        pt.parameterTypes.GroupParameter.__init__(self, name=name, type='group',
```

Comment:
I think the end result is the same, but was there a reason to switch to GroupParameter?

Reply:
Disregard; answered my question already.